esmvalcore/preprocessor/_area.py

```diff
@@ -250,9 +250,22 @@ def area_statistics(cube, operator, fx_variables=None):
     grid_areas = tile_grid_areas(cube, fx_variables)
 
     if not fx_variables and cube.coord('latitude').points.ndim == 2:
-        logger.error(
-            'fx_file needed to calculate grid cell area for irregular grids.')
-        raise iris.exceptions.CoordinateMultiDimError(cube.coord('latitude'))
+        if 'grid_latitude' in [coord.standard_name for coord in cube.coords()]:
```

Comment:
could there be only `grid_latitude` but no `grid_longitude`? I imagine not, but technically you should check for both living in the list

Reply:
Won't be the case but I added it.